CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 
 ### Added
 - regionalisation (#1639)
-- sufficiency scenario (#1642)
+- sufficiency scenario, sufficiency (#1642, #1648)
```

Comment:
Note for reviewers: I put `sufficiency scenario` and `sufficiency` into a single entry to not clutter the change log.